pkg/auth/ldaputil/query.go

```diff
@@ -10,13 +10,34 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/api"
 )
 
+// errNoSuchObject is an error that occurs when trying to find a specific entry fails.
```

Comment:
this looks exactly like `errEntryNotFound`.  In fact, its the same comment.  Why do I have two of these?  defend in the comments or remove.